source/inference.md

```diff
@@ -1257,26 +1052,44 @@ Histograms of first six replicates of bootstrap samples.
 
 +++
 
-We see in {numref}`fig:11-bootstrapping-six-bootstrap-samples` how the
-bootstrap samples differ. We can also calculate the sample mean for each of
-these six replicates.
+We see in {numref}`fig:11-bootstrapping-six-bootstrap-samples` how the distributions of the
+bootstrap samples differ. If we calculate the sample mean for each of
+these six samples, we can see that these are also different between samples.
+To compute the mean for each sample,
+we first group by the "replicate" which is the column containing the sample/replicate number.
+Then we compute the mean of the `price` column and rename it to `sample_mean`
+for it to be more descriptive.
+Finally we use `reset_index` to get the `replicate` values back as a column in the dataframe.
 
 ```{code-cell} ipython3
-six_bootstrap_samples.groupby("replicate")["price"].mean().reset_index().rename(
-    columns={"price": "mean"}
+(
+    six_bootstrap_samples
+    .groupby("replicate")
+    ["price"]
+    .mean()
+    .rename(columns={"price": "sample_mean"})
```

Comment:
@joelostblom I'm seeing a stack trace here and another nearby (which then causes more downstream errors)

Please fix + verify and let me know!

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[39], line 2
      1 (
----> 2     six_bootstrap_samples
      3     .groupby("replicate")
      4     ["price"]
      5     .mean()
      6     .rename(columns={"price": "sample_mean"})
      7     .reset_index()
      8 )

TypeError: Series.rename() got an unexpected keyword argument 'columns'
```

Reply:
(aside from that the PR looks good!)

Reply:
Woops, I swapped the two last lines and forgot to swap them back, fixed.